utils/mdldec/qc.c

```diff
@@ -197,43 +198,67 @@ static void WriteTextureRenderMode( FILE *fp )
 {
 	int		  i;
 	mstudiotexture_t *texture;
+	qboolean	  addnewline = false;
```

Comment:
What if you use ftell here? Let's say:

```
long pos = ftell( fp );
... 
if( ftell != pos )
   fputs( fp, "\n" );
```

Reply:
for what?

Reply:
So you don't have to add `addnewline = true` to every branch here.

Reply:
how?

Reply:
How what? You already added it.

Reply:
Oh... I understood...

Reply:
Isn't it easier? %)))